edx_lint/pylint/unicode_check.py

```diff
@@ -21,10 +21,7 @@ def register_checkers(linter):
 @check_visitors
 class UnicodeFormatStringChecker(BaseTokenChecker):
     """
-    XXXXXX
-
-    The message id is xxxxxxx.
-
+    Checks that strings are unicode.
```

Comment:
Should probably also state the actual ID for the message, as suggested by the prior placeholder template.